packages/nest/src/utils/versions.ts

```diff
@@ -1,8 +1,12 @@
 export const nxVersion = '*';
 
-export const nestJsVersion = '^7.0.0';
-export const nestJsSchematicsVersion = '^7.0.0';
+export const nestJsVersion7 = '^7.0.0';
+export const nestJsVersion8 = '^8.0.0';
 
-export const rxjsVersion = '~6.6.3';
+export const nestJsSchematicsVersion7 = '^7.0.0';
+export const nestJsSchematicsVersion8 = '^8.0.0';
```

Comment:
suggestion/question: this is more of a discussion.

Since we set `@nestjs/schematics` to v8 in `nrwl/nest` `package.json`, should we always force `@nestjs/schematics` to v8 regardless?

I tested and `@nestjs/schematics` does not depend on `rxjs` or anything. The problem with `@nestjs/schematics` staying on v7 is when Angular 13 is part of the migration of the next Nx release, then `@nestjs/schematics@7` will fail because of failed schema validation (due to Angular 13 forcing the `$id` in the schematics schema)

Reply:
agree, i check the @nestjs/schematics dependencies and it seems to be safe to use v8 in both cases